openff/toolkit/topology/molecule.py

```diff
@@ -522,37 +521,15 @@ def molecule_atom_index(self):
         self._molecule_atom_index = self._molecule.atoms.index(self)
         return self._molecule_atom_index
 
-    @property
-    def molecule_particle_index(self):
-        """
-        The index of this Particle within the the list of particles in the parent ``Molecule``.
-        Note that this can be different from ``molecule_atom_index``.
-
-        """
-        if self._molecule is None:
-            raise ValueError("This Atom does not belong to a Molecule object")
-        return self._molecule.particles.index(self)
-
-    # ## From Jeff: Not sure if we actually need this
-    # @property
-    # def topology_atom_index(self):
-    #     """
-    #     The index of this Atom within the the list of atoms in ``Topology``.
-    #     Note that this can be different from ``particle_index``.
-    #
-    #     """
-    #     if self._topology is None:
-    #         raise ValueError('This Atom does not belong to a Topology object')
-    #     # TODO: This will be slow; can we cache this and update it only when needed?
-    #     #       Deleting atoms/molecules in the Topology would have to invalidate the cached index.
-    #     return self._topology.atoms.index(self)
+    # I can deprecate this instead of removing it ... but it seems incredibly unlikely to be used directly
+    # as part of the public API
```

Comment:
Agreed. A GitHub search for this comes up blank, so I don't think we need a deprecation warning. 

```suggestion
```